core/server/services/slack.js

```diff
@@ -48,11 +54,51 @@ function ping(post) {
             return;
         }
 
+        var blogUrl = config.get('url');
+        var blogTitle = settingsCache.get('title');
+
         slackData = {
-            text: message,
+            // We are handling the case of test notification here by checking
+            // if it is a post or a test message to check webhook working.
+            text: schema.isPost(post) ? `Notification from *${blogTitle}* :ghost:` : message,
             unfurl_links: true,
             icon_url: imageLib.blogIcon.getIconUrl(true),
-            username: 'Ghost'
+            username: 'Ghost',
+            // We don't want to send attachment if it is a test notification.
+            attachments: schema.isPost(post) ? [
+                {
+                    fallback: 'Sorry, content cannot be shown.',
+                    title: title,
+                    title_link: message,
+                    author_name: blogTitle,
```

Comment:
A bit confused about why blog title is used as an `author_name` here  We have an actual author in `author` variable which could be used, also from looking into Slack (docs)[https://api.slack.com/docs/message-attachments#author_parameters] we could add an `author_url` linking to authors page :wink:

Reply:
dismiss the comment above, because author related info is in the next section :smile:  wondering though, if having separate section is better then following slacks convention of using `author parameters` :thinking: